exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -345,6 +348,31 @@ void createBlockWithTransactions() {
     }
   }
 
+  @Test
+  void findTransactionsInPool() {
+    try (TestKit testKit = TestKit.forService(TestServiceModule.class)) {
+      TestService service = testKit.getService(TestService.SERVICE_ID, TestService.class);
+
+      String messagePayload = "Test message";
+      TransactionMessage message = constructTestTransactionMessage(messagePayload);
+      RawTransaction rawTransaction = TestKit.toRawTransaction(message);
+      TransactionMessage message2 = constructTestTransactionMessage("Test message 2");
+      RawTransaction rawTransaction2 = TestKit.toRawTransaction(message2);
+
+      try {
+        service.getNode().submitTransaction(rawTransaction);
+        service.getNode().submitTransaction(rawTransaction2);
+      } catch (InternalServerError e) {
+        fail(e);
+      }
+
+      List<TransactionMessage> transactionsInPool =
+          testKit.findTransactionsInPool(
+              tx -> Arrays.equals(tx.getPayload(), messagePayload.getBytes(BODY_CHARSET)));
```

Comment:
Would be shorter:

```suggestion
              tx -> Arrays.equals(tx.getPayload(), message.getPayload()));
```

Or maybe it will be simpler to use different ids?